### ISSUE TITLE
Why <div> and not <ol>

### ISSUE THREAD

**Issue by codepo8** [atom/highlights#17]
That would give us line numbering for free.

**Comment by kevinsawicki** [comment]
`<div>` was used originally to match Atom's HTML structure since it was important to allow themes to work there without changes. And since Atom has a separate gutter component, `<div>` is used there for all lines.

**Comment by codepo8** [comment]
Ah OK, that makes sense. Well, no, but it is a good reason :)

**Comment by kevinsawicki** [comment]
Yeah, I'd be totally up for making this an option and having some sort of line number option available in this library.

**Comment by codepo8** [comment]
well, you can do a CSS counter on the DIV element and generate the numbers
with CSS.
http://christianheilmann.com/2014/11/19/simple-things-styling-ordered-lists/
